salt/modules/openvswitch.py

```diff
@@ -0,0 +1,302 @@
+# -*- coding: utf-8 -*-
+"""
+Support for Open vSwitch
+"""
+
+# Import python libs
+import logging
+
+# Import salt libs
+import salt.utils
+
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    """
+    Only load the module if Open vSwitch is installed
+    """
+    cmd = _detect_os()
+    if salt.utils.which(cmd):
+        return 'openvswitch'
+    return False
+
+
+def _detect_os():
+    """
+    Open vSwitch command may differ depending on packaging.
+    """
+    return 'ovs-vsctl'
+
+
+def _param_may_exist(may_exist):
```

Comment:
Is the overhead of a helper function really necessary here? I'm not sure I follow the need for this.